lib/generators/cosuka_pi_count/cosuka_pi_count_generator.rb

```diff
@@ -10,6 +10,7 @@ def create_initializer_file
         CosukaPiCount.configure do |config|
           config.name = "#{Rails.application.class.module_parent.name.downcase}"
           config.type = 'standard'
+          config.platform = 'aws'
```

Comment:
typeもそうだけど何を入力可能なのかコメントあるほうが良さそう。
↓の中身は適当です。（何が選べるんだっけ、、）

```suggestion
          config.type = 'standard' # `standard` or `xxx`
          config.platform = 'aws'# `aws`, `heroku` or `firebase`
```